src/BinaryBuilderBase.jl

```diff
@@ -17,7 +17,7 @@ export AbstractSource, AbstractDependency, SetupSource, PatchSource,
     generate_compiler_wrappers!, preferred_runner, CompilerShard, UserNSRunner,
     DockerRunner, choose_shards, exeext, preferred_libgfortran_version,
     preferred_cxxstring_abi, gcc_version, available_gcc_builds, getversion,
-    getpkg, aatriplet,
+    getpkg, aatriplet, getcompat,
```

Comment:
Should this even be exported? I only did it for the test suite, but that could use it qualified (or we could even get rid of it, and just directly access `dep.compat`.

Reply:
Yeah, let's export it.  We're pretty liberal with what BBB exports since it's mostly just stuff that used to be in BB.  :P